src/components/fabSpeedDial/demoModal/style.scss

```diff
@@ -0,0 +1,16 @@
+.md-fab.md-mini {
+  background-color: #aaa !important;
```

Comment:
I think it's sufficient to use `.md-button.md-fab`.

--
But it's not that important as long as it works.
The idea is to avoid `!important`, not because it is likely to cause us trouble (the demos are pretty simple apps), but because people tend to copy-paste demo code, so we don't want to promote bad practices :smiley:

So, as long as it works, it's fine by me (simpler is better of course :))